ajax/libs/react-uwp/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "react-uwp",
+  "description": "UWP Design & Fluent Design UiKit by React",
+  "filename": "css/segoe-mdl2-assets.css",
+  "keywords": [
+    "uwp",
+    "microsoft-uwp",
+    "fluent",
+    "microsoft-fluent",
+    "uwp-design",
+    "fluent-design",
+    "react",
+    "typescript",
+    "javascript",
+    "react-components"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/myxvisual/react-uwp.git"
+  },
+  "author": "myxvisual",
+  "license": "MIT",
+  "homepage": "https://www.react-uwp.com",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/myxvisual/react-uwp-bower.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": ["**/*.+(js|css|svg|ttf|woff|woff2|eot|png|jp?eg)"]
```

Comment:
`jp?eg` this part is a little bit weird. Maybe we could use `jpeg|jpg` to match the patterns.

Would you modify from A to B?
- A
```
    "fileMap": [
      {
        "basePath": "",
        "files": ["**/*.+(js|css|svg|ttf|woff|woff2|eot|png|jp?eg)"]
      }
    ]
```

- B
```
    "fileMap": [
      {
        "basePath": "",
        "files": [
          "**/*.+(js|css|svg|ttf|woff|woff2|eot|png|jpeg|jpg)"
        ]
      }
    ]
```

Thank you.